### PR TITLE
Add docs see https://auroraenergyresearch.github.io/aurora-amun-python-sdk-docs/

### PR DESCRIPTION
**Pull request by gareth-rylance-aurora**

### CODE REVIEW

**Comment by gareth-rylance-aurora** [comment]
See https://auroraenergyresearch.github.io/aurora-amun-python-sdk-docs/

File: aurora/amun/client/parameters.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        self.numberOfTurbines = numberOfTurbines
+        self.roughnessLength = roughnessLength
+        self.usePowerCurveSmoothing = usePowerCurveSmoothing
+
+
+class FlowParameters:
+    """Base class for a flow. Internal Use only. 
+
+        Args:
+            windType (str): All flows must define a unique windtype as defined by the API http API.
```
Comment (frisbee09) [inline aurora/amun/client/parameters.py:53]: > API http API

File: aurora/amun/client/parameters.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        self.windType = windType
+
+
+class BuiltInWindParameters(FlowParameters):
+    """The parameters used for built in wind calculations (*era5*,*merra2*,*newa*). Not all locations support all wind types and not all locations support Regional Reanalysis Correction.
+
+        Args:
+            windType (str): one of ("era5","merra2","newa")
+            useReanalysisCorrection (bool, optional):Should Regional Reanalysis Correction be enabled if true then a location specific *reanalysisScaleCorrectionDelta* is used. Defaults to None.
+            reanalysisScaleCorrectionDelta (float, optional): Override the location specific *reanalysisScaleCorrectionDelta*. This has no effect if *reanalysisScaleCorrectionDelta* is false. Defaults to None.
```
Comment (frisbee09) [inline aurora/amun/client/parameters.py:67]: We should document that if nothing is supplied and `useReanalysisCorrection` is on, a location specific default is used. You may want to take a look at the "information" buttons on the UI and they're product owner vetted strings and mean you don't need to type so much :D

File: aurora/amun/client/session.py [hunk 3]
```diff
@@ -121,28 +123,131 @@ def _parse_as_json(self, response):
...
+
         url = f"{self.base_url}/turbines"
         return self._get_request(url)
 
     def get_scenarios(self, region):
-        payload = {}
+        """Get the scenarios that are available for the specified region.
+
+        Args:
+            region (str): The code for the region to lookup scenarios for.
```
Comment (frisbee09) [inline aurora/amun/client/session.py:173]: Do we need to mention all the possible regions? Do we mention that you find your region by asking for details about a location?

File: examples/get_load_factors_example.py [hunk 4]
```diff
@@ -40,39 +49,61 @@ def get_turbine_by_name(turbines, turbine_name):
...
+
+    load_factors = session.run_load_factor_for_parameters(
+        flow_parameters, base_parameters
+    )
+
     loadFactorRequestId = load_factors["parameters"]["loadFactorRequestId"]
     log.info(f"Got result for {loadFactorRequestId}")
     save_to_json(
-        f"load_factors_{datetime.now().isoformat().replace(':','_')}_{loadFactorRequestId}.json",
+        f"load_factors_{datetime.now().isoformat().replace(':','_')}_{flow_parameters.windType}_{loadFactorRequestId}.json",
```
Comment (frisbee09) [inline examples/get_load_factors_example.py:64]: I know this is just an example, but I'm wondering if we do something to make it more obvious that this is the file name that gets spit out.

```
file_name = f"load_factors_{datetime...."
save_to_json(file_name, ...)
```

You and I both know "example" is just the first thing that gets copy/pasted! :D

File: aurora/amun/client/session.py (hunk 3 above)
Comment (gareth-rylance-aurora) [reply aurora/amun/client/session.py:173]: Yes but that is not implemented in wrapper yet!

**Pull request merged**